docs/pages/database-access/enroll-google-cloud-databases/postgres-cloudsql.mdx

```diff
@@ -11,10 +11,10 @@ videoBanner: br9LZ3ZXqCk
 (!docs/pages/includes/database-access/how-it-works/iam.mdx db="PostgreSQL" cloud="Google Cloud"!)
 
 <Tabs>
-<TabItem scope={["oss", "enterprise"]} label="Self-Hosted">
+<TabItem label="Self-Hosted">
 ![Teleport Database Access CloudSQL Self-Hosted](../../../img/database-access/guides/cloudsql_selfhosted.png)
 </TabItem>
-<TabItem scope={["cloud"]} label="Teleport Enterprise Cloud">
+<TabItem label="Teleport Enterprise Cloud">
```

Comment:
In general, we're steering the docs toward using "Teleport Enterprise" for both cloud-hosted and self-hosted Enterprise accounts, and using some other way (other than the title of the product) to clarify that it's hosted in the cloud or self hosted.

Reply:
done: 3f3e69cdf5aa4afcdbdf42cceba16d3e819566c1

Reply:
(see slack thread): I wonder if I shouldn't use this phrasing? It makes it sound like self-hosted is distinct from enterprise

Reply:
Switching this to "Cloud-Hosted" now based on discussion in another PR